src/ucp/proto/proto_multi.inl

```diff
@@ -189,14 +189,14 @@ static UCS_F_ALWAYS_INLINE ucs_status_t ucp_proto_multi_zcopy_progress(
         ucp_proto_init_cb_t init_func, unsigned uct_mem_flags, unsigned dt_mask,
         ucp_proto_send_multi_cb_t send_func,
         ucp_proto_complete_cb_t complete_func,
-        uct_completion_callback_t uct_comp_cb)
+        uct_completion_callback_t uct_comp_cb,
+        ucp_proto_request_zcopy_init_cb_t zcopy_init)
```

Comment:
why can't use 'ucp_proto_init_cb_t init_func' ?
maybe consolidate init_func and zcopy_init tp single argument?